packages/lib/src/project/BasePackageProject.ts

```diff
@@ -74,7 +76,12 @@ export default abstract class BasePackageProject {
     );
     const directory: ImplementationDirectory = await this.getCurrentDirectory();
     await directory.unsetImplementation(contractName);
-    Loggy.onVerbose(`unset-implementation-${contractName}`);
+    Loggy.onVerbose(
```

Comment:
Shouldn't we keep this one as it was, but change the `onVerbose` to `succeed`?
It's not a big deal though, as this will be logged only on verbose mode, but will be consistent with other logs throughout the project.

Reply:
Your call! Does Logger support calling a `success` on a `onVerbose`? If so, I'll adjust this.